README.md

```diff
@@ -50,9 +50,9 @@ Load the CKEditor 4 script from CDN:
 </script>
 ```
 
-### Integrating with Angular and React
+### Integrating with Angular, React and Vue.js
```

Comment:
TBH I'm not sure if it should be `Vue.js` or just `Vue`? In most places in docs we mention it as `Vue` (except for https://ckeditor.com/docs/ckeditor4/latest/guide/index.html where `Vue.js` is used) :thinking: 

Looking at https://www.npmjs.com/package/vue it seems official project name is `Vue.js` (while package is simply named `vue`). So the question is if we should keep it consistent or have correct name here :see\_no\_evil:

Reply:
Ok, looking more in some other places it seems using `Vue.js` is a correct approach.

Reply:
Yeah, I wasn't sure about it too, but seems they are adding `.js` as a part of official name 🤔